### DISCUSSION TITLE
mmsi lookup

### DISCUSSION THREAD

**Discussion by cscompton** [jvde-github/AIS-catcher#197] [Q&A] [answered]
I noticed the map for AIS-Catcher looks up and displays the ships name on the map. This information isn't stored/forwarded in the ais data. Does AIS-Catcher output a json file anywhere that I can query to pull in ship information? I'm forwarding the AIS data to Node-Red and mapping the ship on a worldmap. If AIS-Catcher isn't generating json that I can query, does anyone know a easy way to lookup the mmsi and return the ships name in node-red?

**Comment by cscompton** [comment, marked as answer]
using dev tools in chrome I found this you can poll however you want and it stores the ships name. http://ipaddress:8383/ships_array.json

**Reply by jvde-github** [reply]
Correct. You can also download ships.json, same data but still contains the field names, if easier.
